04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -162,35 +167,47 @@ box(col="white")
 
 ```
 
-```{r reset-par, echo=FALSE, results="hide"}
-#making sure this doesn't render in the lesson
+In a multi-band dataset, the rasters will always have the same extent,
+CRS and resolution.  
+
+```{r reset-par, echo=FALSE, results="hide", warning=FALSE}
 par(original_par) # go back to original par
 
 ```
 
+Want to learn more? The video below, covers the basics of what a multi-band 
+images are:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/3iaFzafWJQE" frameborder="0" allowfullscreen></iframe>
 
 #Other Types of Multi-band Raster Data
 
 Multi-band raster data might also contain:
 
-1. Time series: same variable, over the same area, over time. 
-[Check out Lesson 04 - Raster time series data in R to learn more about time series stacks]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/).
-2. Multi-hyperspectral imagery: that might have 4 or more bands up to 400+ bands 
-of image data! Check out a lesson on 
+1. Time series: same variable, over the same area, over time. Check out 
+[Raster Time Series Data in R ]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/) to learn more about time series stacks.
+2. Multi-hyperspectral imagery: rasters that have 4 or more (even 400+) bands 
+image data! Check out the 
 <a href="http://neondataskills.org/HDF5/Imaging-Spectroscopy-HDF5-In-R/" target="_blank">
-hyperspectral imagery here.</a>
+NEON Data Skills Imaging Spectroscopy HDF6 in R lesson.</a>
```

Comment:
HAHAHA - this was another good chuckle for me! I love it. The next super generation :)